lib/models/subscription.js

```diff
@@ -119,6 +119,20 @@ module.exports = class Subscription extends Sequelize.Model {
     return results
   }
 
+  static async agedSyncCounts () {
```

Comment:
should we add a comment here to note that if you change the `interval` you need to change the metric name too?

Reply:
Hrm. I think paramaterizing the day probably makes more sense. Then the "7" will live only in the job code. I'll hit that in the followup too.